### PR TITLE
[13.0][ADD] hr_study: New module

### PR DESCRIPTION
**Pull request by victoralmau**
New module

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT41971

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Test and code review :+1: 

Should studies be linked to a certification level value?

File: hr_study/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_study
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows your to define employee studies.
```
Comment (pedrobaeza) [inline hr_study/README.rst]:
```suggestion
This module allows your to define employee studies in a structured way. Previously, the standard only allows to define a free text for this field, getting no homogenized values.
```

File: hr_study/README.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Configuration
+=============
+
+You can define the studies from Employees > Configuration > Studies
+
+Usage
+=====
+
+Go to an employee's file, and in the "Education" block ("Private information" tab) you can set a study.
```
Comment (pedrobaeza) [inline hr_study/README.rst:43]: Add in ROADMAP to auto-include a "Other" record, that shows and allows to set the free text.

File: hr_study/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Hr Study",
```
Comment (pedrobaeza) [inline hr_study/__manifest__.py]:
```suggestion
    "name": "Employees education",
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I think a better name would be `hr_education`. What do you think?

**Comment by chienandalu** [comment]
> I think a better name would be `hr_education`. What do you think?

My first impression was similar, but then I saw that the original text field was called like that.

**Comment by pedrobaeza** [comment]
> Should studies be linked to a certification level value?

We can put this in roadmap, but this is usually something different.

**Comment by victoralmau** [comment]
Changes done. I think the name `hr_education` could cause confusion as it is not strictly linked.

**Comment by pedrobaeza** [comment]
Well, I think it is linked. Maybe `hr_education_level`? Or what is the purpose of our customer with this field?

**Comment by victoralmau** [comment]
> Well, I think it is linked. Maybe `hr_education_level`? Or what is the purpose of our customer with this field?

The purpose is to avoid writing always the same thing (and possible errors) by being able to select possible values from a previously defined list.

**Comment by pedrobaeza** [comment]
I don't mean that. I mean that what is the information that they store there.

**Comment by victoralmau** [comment]
> I don't mean that. I mean that what is the information that they store there.

Some information that may serve as a reference:

Employee 1:
`certificate`: master
`study_field`: Law

Employee 2:
`certificate`: other
`stdy_field`: Senior technician in computer applications development

**Comment by pedrobaeza** [comment]
OK, so it already exists another field for what I call "study level". Uf, the terminology used by Odoo is a bit confused, but let's go then with current name.

File: hr_study/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Employees education",
```
Comment (pedrobaeza) [inline hr_study/__manifest__.py]: With the latest information, this should be

```suggestion
    "name": "Employees study field",
    "summary": "Structured study field for employees",
```

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**